src/OrchardCore.Modules/OrchardCore.Taxonomies/Drivers/TaxonomyContentsAdminListDisplayDriver.cs

```diff
@@ -156,6 +158,60 @@ private static void PopulateTermEntries(List<FilterTermEntry> termEntries, IEnum
                     PopulateTermEntries(termEntries, children, level + 1);
                 }
             }
+        }
+               private static string GetDifferentiator(ContentItem contentItem)
+        {
+            var identifier = contentItem.Content.AliasPart?.Alias?.ToString() ?? contentItem.DisplayText ?? contentItem.ContentItemId;
+            var encodedIdentifier = EncodeAlternateElement(identifier);
+            var differentiator = FormatName(encodedIdentifier);
```

Comment:
Just `FormatName` - `EncodeAlternate` is for if we're making shape alternates.

Formatting too ;)